### ISSUE TITLE
Allow more fine grained OAuth setup

### ISSUE THREAD

**Issue by paul-hammant** [prose/prose#1135]
![image](https://user-images.githubusercontent.com/82182/224529265-964edccf-f712-4cc0-aed3-da4eb6b2b1e7.png)

It would be great to pick orgs and/or pick repos .. subsetting from a much larger set.

**Comment by TheBigRoomXXL** [comment]
Yes! For me that much access is too much. I should at least be able to disable access to organization I am part of. Giving too much access to my personal repos is a my problem but giving access to other people repos is another thing entirely.

A good example is Netlify, it support very fined grained access, I can choose exactly which repos I want to give access.

**Comment by babbitt** [comment]
Unfortunately for now this is something outside of the realm of being able to be fixed by me. Although I have admin permissions for this repo, I don't have access to the Oauth app that powers prose (that's owned by one of the former maintainers). 
Keeping this open in case they ever come back but I wouldn't hold your breath that this is something we can tackle soon.
